TokenService.js

```diff
@@ -60,8 +60,8 @@ export class TokenService {
     payload.authenticationMethod = authenticationMethod;
     payload.requiredAuthenticationMethods = requiredAuthenticationMethods;
 
-    await axios.post(url + `/${type}`, payload);
-    return payload;
+    const response = await axios.post(url + `/${type}`, payload);
```

Comment:
Need a serviceId in the payload from somewhere?

Reply:
+1